File: sennse_ar_app/src/app/model/board.model.ts
import { Sensor } from "./sensor.model";

export interface Board{
    id: number;
    boardName: string;
    boardDescription: string;
    boardUUId: string;
    qrCodePath: string;
    targetIndex?: number;
    sensorModelList: Sensor[];
    sensorHandled?: Array<Record<string, string>>;
}

File: SENNSE-AR-MS-SB/src/main/java/sennse/model/BoardModel.java
package sennse.model;

import lombok.Data;

import java.util.HashMap;
import java.util.List;
import java.util.UUID;

@Data
public class BoardModel {
    private int id;

    private String boardName;

    private String boardDescription;

    private UUID boardUUId;

    private String qrCodePath;

    private List<HashMap<String, String>> sensorHandled ;

    private List<SensorModel> sensorModelList;

}
